pom.xml

```diff
@@ -169,6 +169,7 @@
     <!-- PHOENIX-6806 - Default protoc architecture to the detected one,
       but allow for profile overrides -->
     <protoc.arch>${os.detected.classifier}</protoc.arch>
+    <glassfish.el.version>3.0.1-b08</glassfish.el.version>
```

Comment:
Maybe good to add a comment here "keep the version in sync with hbase pom's javax.el version"?